NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -790,7 +806,8 @@ void NeuralAmpModeler::_ProcessInput(iplug::sample** inputs, const size_t nFrame
 void NeuralAmpModeler::_ProcessOutput(iplug::sample** inputs, iplug::sample** outputs, const size_t nFrames,
                                       const size_t nChansIn, const size_t nChansOut)
 {
-  const double gain = pow(10.0, GetParam(kOutputLevel)->Value() / 20.0);
+  const bool namActive = GetParam(kNamToggle)->Value();
```

Comment:
[Nit] I don't like that we `GtParam(kNamToggle)->Value()` twice in the RTL. Perhaps a variable could be assigned "`OnParamChange`".

Reply:
Good point, fixed.